google-login-plugin/src/com/google/gct/login/stats/UsageTrackerServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gct.login.stats;
+
+import com.intellij.openapi.util.KeyedExtensionCollector;
+import com.intellij.util.PlatformUtils;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Usage Tracker Service for obtaining extensions declared in plugin.xml with {@link UsageTrackerExtensionPointBean}.
+ */
+public final class UsageTrackerServiceImpl extends UsageTrackerService {
```

Comment:
I'm not a big fan of ...Impl class names.  if the API was called UsageTrackerProvider this implementation could be ExtensionPointBasedUsageTrackerProvider or something like that.